server/model.js

```diff
@@ -1,6 +1,70 @@
 const client = require('./db.js');
 
 module.exports = {
+  findUserType: (uuid) => {
+    const queryString = `SELECT account_type FROM "Firebase" WHERE user_uuid = ${uuid}`;
+    return client.query(queryString);
+  },
 
+  addSeeker: (seeker) => {
+    const { user_uuid, first_name, last_name, coord_lat, coord_long, pref_industry, resume_url } = seeker;
+    const queryString = `INSERT INTO "Seekers" VALUES ('${user_uuid}', '${first_name}', '${last_name}', ${coord_lat}, ${coord_long}, '${pref_industry}', '${resume_url}')`;
+    return client.query(queryString);
+  },
 
+  addRecruiter: (recruiter) => {
+    const { user_uuid, first_name, last_name, company_name } = recruiter;
+    const queryString = `INSERT INTO "Recruiters" VALUES ('${user_uuid}', '${first_name}', '${last_name}', '${company_name}')`;
+    return client.query(queryString);
+  },
```

Comment:
for the addSeeker and addRecruiter, are you going to post the uuid and account_type to the Firebase table also?

Reply:
Thanks a bunch for catching that!